create/how_to/aws.md

```diff
@@ -0,0 +1,181 @@
+
+# Deploy a MeiliSearch Instance on AWS
+
+Using our **MeiliSearch [AMI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)**, MeiliSearch can be deployed on AWS in just a few minutes.
+The following `how-to guide` will walk you through every step to deploy MeiliSearch in an AWS EC2 instance.
+
+**Table of Contents**:
+
+[[toc]]
```

Comment:
There is something weird, but it may not be just there, it may be on all the site. The permalink pointing to achors are working when you click them inside the page, but if you open a link on a new tab, it doesn't work. You can also see the issue if you copy a link pointing to an anchor, and paste it yourself in your browser, it will open the right page, but not scroll to the right position.

Reply:
I think this is a couple different behaviors.

1. If you select a header smaller than an `<h2>` the anchor link automatically changes to the parent header. I assume this is a VuePress thing but no idea if it's modifiable or not.
2.  If you open a page with lots of images (like our deploy guides), the page will scroll to the anchor BEFORE the images load. Then, when the images load in that pushes the content further down and you end up looking at the wrong place on the page.

Does that explain/cover the behavior you're seeing or is there another aspect to it? And do you see the same behavior on both the live site and the deploy previews?

Reply:
No, it's a new one! Let me do a video :)

Reply:
Or maybe it's the same issue, but check. The fact that VuePress cannot manage something as basic as links and anchors is another reason for switching IMNSHO

https://user-images.githubusercontent.com/1170989/108264838-06220e80-7136-11eb-961c-c88bc9ce948d.mp4

Reply:
Anyway, it's a problem with VuePress, not this PR.